### DISCUSSION TITLE
ERROR: Unable to start debugging. Unexpected GDB output from command "-exec-run". Don't know how to run. Try "help target".

### DISCUSSION THREAD

**Discussion by ElenaG391** [community/community#127088] [Programming Help]
### Select Topic Area

Question

### Body

+ Ubuntu 22.04 
+ using avr-gdb (GDB version 12.1)
+ avr-gcc (gcc version 11.4)
+ run my program and the file that generate is pcl_digtest.elf

elena@Elena:~/Documents/1288/pclqube$ file pcl_digtest.elf
pcl_digtest.elf: ELF 32-bit LSB executable, Atmel AVR 8-bit, version 1 (SYSV), statically linked, with debug_info, not stripped

When I try to use GDB to debug the program I receive this error:

ERROR: Unable to start debugging. Unexpected GDB output from command "-exec-run". Don't know how to run.  Try "help target".
The program '/home/elena/Documents/1288/pclqube/pcl_digtest.elf' has exited with code 42 (0x0000002a).

any idea?

thanks

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @ElenaG391 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by ElenaG391** [reply]
thanks!

**Comment by queenofcorgis** [comment]
Hey there @ElenaG391 👋🏼 

We're going through some unanswered posts, were you able to get this solved?
